Src/Generated/Google.Apis.Discovery.v1/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": "1.15.0",
+
+  "dependencies": {
+    "GoogleApis_dotnetcore": { "target": "project" },
```

Comment:
Nothing special. It's just a convention in this repo that a _dotnetcore suffix is used for the directory that contains the dotnet cli project.json files. I expect it'll go away when we move the build to only use dotnet cli.